plugins/dai-release-backend/src/service/router.ts

```diff
@@ -188,6 +193,52 @@ export async function createRouter(
     res.status(200).json(metaInformation);
   });
 
+  router.post('/workflows', async (req, res) => {
+      if (permissions && httpAuth) {
+        const decision = await permissions.authorize(
+          [{ permission: daiReleaseViewPermission }],
+          { credentials: await httpAuth.credentials(req) },
+        );
+        const { result } = decision[0];
+        if (result === AuthorizeResult.DENY) {
+          throw new NotAllowedError(
+            'Access Denied: Unauthorized to access the Backstage Release plugin',
+          );
+        }
+      }
+    const instanceName = req.query.instanceName?.toString() || '';
+    const pageNumber = getEncodedQueryVal(req.query.pageNumber?.toString());
+    const resultsPerPage = getEncodedQueryVal(
+      req.query.resultsPerPage?.toString(),
+    );
+    const searchInput = req.query.searchInput?.toString() || '';
+    const categories: string[] = req.query.categories
+      ? req.query.categories.toString().split(',')
+      : [];
+    const author = req.query.author?.toString() || '';
+    const workflows = await workflowsOverviewApi.getWorkflowsOverviewApi(instanceName, pageNumber, resultsPerPage, searchInput, categories, author);
+    res.status(200).json(workflows);
+  });
+
+  router.post('/redirect-to-workflow', async (req, res) => {
```

Comment:
The Id we get from the list is below, if fronend
 needs to send in the URL , then need to just send the "Release2bb84833587a48bf8af3943006e1acdf" instead of whole thing.

Applications/FolderDefaultReleaseContent/Folder0a5f467c12cf41ce967092077b2138e5/Folder303182ca1d5443b2b63a0ff04eec5878/Release2bb84833587a48bf8af3943006e1acdf

Thats why thought of sending this as a body as templateId: "Applications/FolderDefaultReleaseContent/Folder0a5f467c12cf41ce967092077b2138e5/Folder303182ca1d5443b2b63a0ff04eec5878/Release2bb84833587a48bf8af3943006e1acdf"

Reply:
Made the required changes, changed into /workflow/redirect.